.travis.yml

```diff
@@ -8,32 +8,38 @@ matrix:
   include:
     - os: linux
       compiler: gcc
-      env: BUILD_TYPE=Debug COVERALLS=ON COMPILER=GCC
+      env: BUILD_TYPE=Debug COVERALLS=ON COMPILER=GCC LLVM_VERSION=3.6
     - os: linux
       compiler: gcc
-      env: BUILD_TYPE=Release COVERALLS=OFF COMPILER=GCC
+      env: BUILD_TYPE=Release COVERALLS=OFF COMPILER=GCC LLVM_VERSION=3.6
     - os: linux
       compiler: clang
-      env: BUILD_TYPE=Debug COVERALLS=OFF COMPILER=CLANG
+      env: BUILD_TYPE=Debug COVERALLS=OFF COMPILER=CLANG LLVM_VERSION=3.6
     - os: linux
       compiler: clang
-      env: BUILD_TYPE=Release COVERALLS=OFF COMPILER=CLANG
+      env: BUILD_TYPE=Release COVERALLS=OFF COMPILER=CLANG LLVM_VERSION=3.6
     - os: osx
       compiler: clang
-      env: BUILD_TYPE=Debug COVERALLS=OFF COMPILER=CLANG
+      env: BUILD_TYPE=Debug COVERALLS=OFF COMPILER=CLANG LLVM_VERSION=3.7
     - os: osx
       compiler: clang
-      env: BUILD_TYPE=Release COVERALLS=OFF COMPILER=CLANG
+      env: BUILD_TYPE=Release COVERALLS=OFF COMPILER=CLANG LLVM_VERSION=3.7
+    - os: osx
+      compiler: clang
+      env: BUILD_TYPE=Debug COVERALLS=OFF COMPILER=CLANG LLVM_VERSION=latest
+    - os: osx
+      compiler: clang
+      env: BUILD_TYPE=Release COVERALLS=OFF COMPILER=CLANG LLVM_VERSION=latest
 
 install:
-  - if [ "$TRAVIS_OS_NAME" = "linux" ]; then 'ci/install_linux.sh'; fi
-  - if [ "$TRAVIS_OS_NAME" = "osx"   ]; then 'ci/install_macos.sh'; fi
+  - if [ "${TRAVIS_OS_NAME}" = "linux" ]; then . ci/install_linux.sh; fi
+  - if [ "${TRAVIS_OS_NAME}" = "osx"   ]; then . ci/install_macos.sh; fi
 
 script:
   - mkdir build
   - cd build
-  - if [ "$TRAVIS_OS_NAME" = "linux" ]; then cmake -DCMAKE_BUILD_TYPE=$BUILD_TYPE -DLLVM_DIR="/usr/share/llvm-3.6/cmake/" ..; fi
-  - if [ "$TRAVIS_OS_NAME" = "osx"   ]; then cmake -DCMAKE_BUILD_TYPE=$BUILD_TYPE -DLLVM_DIR="/usr/local/Cellar/llvm@3.7/3.7.1/lib/llvm-3.7/share/llvm/cmake/" ..; fi
+  - if [ "${TRAVIS_OS_NAME}" = "linux" ]; then cmake -DCMAKE_BUILD_TYPE=${BUILD_TYPE} -DLLVM_DIR="/usr/share/llvm-${LLVM_VERSION}/cmake/" ..; fi
+  - if [ "${TRAVIS_OS_NAME}" = "osx"   ]; then cmake -DCMAKE_BUILD_TYPE=${BUILD_TYPE} -DLLVM_DIR="$(brew --prefix llvm@${LLVM_VERSION})/lib/llvm-${LLVM_VERSION}/share/llvm/cmake/" ..; fi
```

Comment:
It seems we should use if-statement here as well to get the correct prefix when `LLVM_VERSION=latest`, or let `ci/install_macos.sh` to set a variable according to the LLVM version like `llvm@3.x` (for 3.6, 3.7, 3.8) or `llvm` (for lastest) so that we can use the variable here.